src/infrastructure/jira/jira-service.ts

```diff
@@ -95,6 +102,11 @@ class JiraService {
 				design,
 				connectInstallation,
 			),
+			this.updateIngestedDesignsIssueProperty(
```

Comment:
@jvrankul23 @achandra-atlas can we discuss this decision again? Having reviewed it, I wonder if comparing the relationships if the better option so that the lazy backfill experience is not dependant on the app doing the right thing.